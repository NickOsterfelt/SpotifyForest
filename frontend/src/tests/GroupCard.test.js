import React from "react";
import { render } from "@testing-library/react";
import GroupCard from "../components/GroupCard";

it("renders without crashing", function() {
  render(<GroupCard />);
});

it("matches snapshot", function() {
  const { asFragment } = render(<GroupCard />);
  expect(asFragment()).toMatchSnapshot();
});
