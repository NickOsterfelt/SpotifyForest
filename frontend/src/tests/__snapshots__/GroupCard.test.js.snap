// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="m-3 card"
      style="width: 18rem;"
    >
      <div
        class="mb-2 text-dark card-body"
      >
        <div
          class="card-title h5"
        />
        <div
          class="mb-2 text-muted card-subtitle h6"
        >
          Number of users: 
        </div>
        <p
          class="card-text"
        >
          Description: 
        </p>
        <button
          class="btn btn-primary btn btn-block"
          type="button"
        >
          Join
          <i
            class="fas fa-plus text-success border bg-light rounded p-1 mx-2"
          />
           
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
